Src/Generated/Google.Apis.ManufacturerCenter.v1/Google.Apis.ManufacturerCenter.v1.cs

```diff
@@ -1554,9 +1554,42 @@ public class Issue : Google.Apis.Requests.IDirectResponseSchema
         [Newtonsoft.Json.JsonPropertyAttribute("severity")]
         public virtual string Severity { get; set; }
 
+        private string _timestampRaw;
+
+        private object _timestamp;
+
         /// <summary>The timestamp when this issue appeared.</summary>
         [Newtonsoft.Json.JsonPropertyAttribute("timestamp")]
```

Comment:
If we were to move this one to TimestampDateTimeOffset (by specifying to use GetDateTimeOffsetFromString to serialize it) would that give us the failure right before performing the request instead of having to wait for the server to do that?

Reply:
I think so - but we'd have to also add a converter (either specified in the property or in general) to make sure that the `DateTimeOffset` is serialized correctly. It would work, but there'd be more risk of problems elsewhere IMO.